README.md

```diff
@@ -0,0 +1,54 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# portfolio project
+
+> this is my first portifolio.
+
+![screenshot](./images/screenshot/screenshot-mobile-version.png)
```

Comment:
- [ ] It would be nice to only have the working version of the project as a screenshot rather than showing the source code as that is unnecessary.